### PR TITLE
Add VEED icon

### LINKED ISSUE #8620: Request: VEED.IO
### Brand Name

VEED.IO

### Website

https://veed.io

### Popularity Metric

Global rank 3313 (https://www.similarweb.com/website/veed.io/#overview)

### Official Resources for Icon and Color

I couldn't found it..

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by uncenter**
image

**Issue:** closes #8620.

**Popularity metric:** Ranked 3,313 on Similarweb. See https://www.similarweb.com/website/veed.io/#overview.

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Color picked from the color of the SVG in the header and background color of the footer. Can't find the green from the favicon anywhere.

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
They use just the `V` in a green colour as their favicon, an SVG for just the 'V' can be seen in their website footer.

**Comment by uncenter** [comment]
> They use just the `V` in a green colour as their favicon, an SVG for just the 'V' can be seen in their website footer.

You had me confused with https://github.com/simple-icons/simple-icons/issues/8620#issuecomment-1640808164! I'll update it to just the V.

**Comment by adamrusted** [comment]
> You had me confused with [#8620 (comment)](https://github.com/simple-icons/simple-icons/issues/8620#issuecomment-1640808164)! I'll update it to just the V.

Looking at the fact they  _were_ referring to themselves as veed.io - not just veed - I would assume they've since rebranded ;)

**Comment by adamrusted** [comment]
Do we want to use their green (seen in socials, and gradients) too? Bit more exciting than the grey!

**Comment by uncenter** [comment]
My color picker can't decide between `#B7FF60` and `#B5FF60` for the green they use across socials. What are you picking up?

**Comment by adamrusted** [comment]
I'm seeing slight variance between the socials, but most hover around `#B6FF60` which is in their Tailwind styles ;)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -16461,6 +16461,11 @@
             "hex": "00B336",
             "source": "https://www.veeam.com/newsroom/veeam-graphics.html"
         },
+        {
+            "title": "VEED",
+            "hex": "323232",
```
Comment (uncenter) [inline _data/simple-icons.json:16466]:
```suggestion
            "hex": "B6FF60",
```

**Review by adamrusted — APPROVED** [review]

**Pull request merged**